src/app/friends/add-edit-friends/add-edit-friends.component.html

```diff
@@ -1 +1,41 @@
-<p>add-edit-friends works!</p>
+<!-- Group Add and Edit Form  -->
+<form [formGroup]="addEditFriend">
+
+    <!-- Form Header  -->
+    <div class="modal-header">
+
+        <!-- If clicked on add button -->
+        <h1 class="modal-title fs-5" id="exampleModalLabel">Invite Friend</h1>
+        <!-- close button -->
+        <button type="button" class="btn-close" data-bs-dismiss="modal">.</button>
+    </div>
+
+    <!-- Form body -->
+    <div class="modal-body">
+
+        <!-- Form Controls -->
+
+        <!-- group name -->
+        <div class="mb-3">
+            <label for="friend-name" class="form-label fs-5">Friends Email: </label>
+            <input type="email" class="form-control" id="friend-name" placeholder="Enter Group Name" formControlName="friendName">
+        </div>
+
+        <!-- message  -->
+        <div class="mb-3">
+            <label for="Textarea" class="form-label fs-5">Message: </label>
+            <div class="form-floating">
+                <textarea class="form-control" placeholder="Leave a comment here" id="Textarea" style="height: 100px" formControlName="message"></textarea>
```

Comment:
1. Remove inline style and use css instead
2. Lowercase 'T' from id Textarea

Reply:
I changed the comment message. 
and changed the id of textarea.